src/app/collective-rewards/rewards/backers/hooks/useGetBackerRewards.ts

```diff
@@ -73,9 +73,9 @@ export const useGetBackerRewards = (
   const rbtcPrice = prices[rbtc.symbol]?.price ?? 0
 
   const data = useMemo(() => {
-    return builders.reduce<BackerRewards[]>((acc, builder) => {
+    return builders.reduce<BackerRewards[]>((acc, builder, i) => {
       const { address, gauge } = builder
-      const backerAllocationOf = allocationOf[gauge] ?? 0n
+      const backerAllocationOf = allocationOf[i] ?? 0n
```

Comment:
Same as for other cases, I think that using the index may be misleading as we already experienced issues with this approach